### ISSUE TITLE
Dropping same file to field between resets doesn't trigger reactivity for field.value

### ISSUE THREAD

**Issue by susickypavel** [fabian-hiller/modular-forms#11]
Hi,

# Reproduction

1. Use either form with input field here https://modularforms.dev/playground/special. 
2. Drop a file to the input
3. Reset the form with the button on the top
4. Try to drop the same file into the input

# Expected behaviour

The name of the file should be again visible in the field.

# Actual behaviour

The name doesn't show. Instead, the label value stays.

# Workaround

I had same problem and one obscure workaround that I found was:

```tsx
<Field ...>
	{field => {
		// Line below solves the problem.
		field.value;

		return (
			<input ... />
		)
	}}
</Field>
```

**Comment by fabian-hiller** [comment]
Hey, the problem here is that `<input type="file" />` fields cannot be easily controlled. I'll have a closer look at that. I suspect I have a possible solution in mind. Thank you for highlighting this. I'll keep you posted here.

**Comment by fabian-hiller** [comment]
Should be fixed with v0.9.3. Thanks for your feedback!
